x/gamm/pool-models/balancer/amm.go

```diff
@@ -258,11 +261,10 @@ func (p *Pool) JoinPool(_ctx sdk.Context, tokensIn sdk.Coins, swapFee sdk.Dec) (
 }
 
 // CalcJoinPoolShares
```

Comment:
We should finish the godoc :P

```suggestion
// CalcJoinPoolShares
```